rotkehlchen/db/upgrades/v35_v36.py

```diff
@@ -43,7 +43,8 @@ def _remove_adex(write_cursor: 'DBCursor') -> None:
         write_cursor.execute('DROP TABLE IF EXISTS adex_events')
         if table_exists(write_cursor, 'used_query_ranges'):
             write_cursor.execute(
-                'DELETE FROM used_query_ranges WHERE name LIKE ?', ('adex_events%',),
+                'DELETE FROM used_query_ranges WHERE name LIKE ? ESCAPE ?',
+                (r'adex\_events%', '\\'),
```

Comment:
This is not incosistent with the rest of the changes. Make it like the rest please.